tf2_geometry_msgs/CMakeLists.txt

```diff
@@ -46,5 +46,5 @@ install(DIRECTORY include/${PROJECT_NAME}/
 #
 #
 # endif()
-ament_export_include_directories(include)
-ament_package()
+ament_auto_find_build_dependencies()
```

Comment:
This line should be above all your targets (like the current `find_package` calls). The existing `find_package()` calls are (almost) obsolete if you don't want to ensure `REQUIRED`. Otherwise I would suggest checking `<pkgname>_FOUND` instead.